src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -102,6 +106,7 @@ function AppointmentsPage({
     <Switch>
       <Route component={PastAppointmentsList} path="/past" />
       <Route component={ExpressCareList} path="/express-care" />
+      <Route component={NewBooking} path="/new-project-cheetah-booking" />
```

Comment:
There's already a route for this in the main routes file. These are just for sub-routes on the appointments page.